scss/_footer.scss

```
@@ -0,0 +1,97 @@
+*, ::before, ::after {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    transition: all .1s ease-in;
+    font-family: "Lato";
```

Не забывайте прописывать базовый шрифт
```suggestion
    font-family: "Lato", sans-serif;
```